ui/app/serializers/database/connection.js

```diff
@@ -29,6 +29,16 @@ export default RESTSerializer.extend({
       ...payload.data,
       ...payload.data.connection_details,
     };
+
+    // connection_details are spread above into the main body of response so we can remove redundant data
+    delete response.connection_details;
```

Comment:
This was some cleanup, but i can remove if folks are worried about regressions. I searched globally for any use of `connection_details` or `connectionDetails` related to databases and didn't come across anything.